resources/guidelines/documentation-guidelines/01-general.md

```diff
@@ -0,0 +1,55 @@
+# Documentation Guidelines
+
+Your style, your words, and your tone define you. Correspondingly, this style guide lays out standard editorial instructions that define Shopware documentation.
+
+We recommend that every contributor follows the defined writing standards to maintain uniformity of the documentation. Refer to different sections of this guide to know more.
+
+## Audience
+
+It is important to consider the background of the potential audience reading your writing. This helps you to adapt your writing to meet their needs and interests.
+
+| Who is the audience? | What are their roles? |
+| :--- | :--- |
+| Fullstack developer | <ul><li>Plugin development</li><li>Templates</li><li>Routes/ Controllers</li></ul>|
+| Frontend developer | <ul><li>Admin</li><li>Themes</li><li>PWA</li></ul>  |
+| Backend developer | <ul><li>DI / Service architecture</li><li>Message queues</li><li>DAL</li><li>Action event system</li><li>ElasticSearch</li></ul> |
+| API developer  | <ul><li>How to consume the API</li><li>Create a product / category import</li><li>How to extend the API</li><li>API paradigm</li><li>Proper API references</li><li>Request collection</li></ul> |
+| DevOps  | <ul><li>Hosting setup</li><li>Deployment</li><li>Performance tests</li></ul> |
+| Project/Solution architect | <ul><li>Hosting</li><li>Architecture</li><li>Modules / Extension system</li><li>Paradigms / patterns</li><li>Commonalities app system / Plugin system</li></ul> |
+| Designer | <ul><li>Component library</li><li>Design system</li></ul> |
+| Product owner/ manager | <ul><li>Responsibilities pertaining to the product life cycle</li></ul> |
+| Tech writers | <ul><li>Document all product details</li></ul> |
+
+## List of docs maintained by us
+
+* [PWA](https://shopware-pwa-docs.vuestorefront.io/)
+
+* [Developer Guide](https://developer.shopware.com/docs/)
+
+* [API Reference Guide](https://shopware.stoplight.io/)
+
+* [Admin Extension SDK](https://shopware.github.io/admin-extension-sdk/)
+
+## Word list
+
+Choose ecommerce and technical terms from the pre-defined list of terminologies:
```

Comment:
We can add a hint here, indicating that these are internal resources only visible to Shopware employees.